docs/rate-limits.md

```diff
@@ -63,14 +63,17 @@ renewals of existing certificates.
 Revoking certificates does not reset rate limits, because the resources used to
 issue those certificates have already been consumed.
 
-We have also recently (December 2016) introduced a rate limit on overall request
-volume. The "new-reg", "new-authz" and "new-cert" endpoints have an **Overall
-Request Per Second** rate limit of 20 requests per second. All other endpoints
-have a rate limit of 2000 requests per second.
+We will soon (January 2016) introduce a **Failed Validation** limit of 5 failures
+per account, per hostname, per hour. This limit will be higher on staging so you
+can use staging to debug connectivity problems.
```

Comment:
Could you add a mention of the higher limit in the [rate limits section of the staging page](https://github.com/letsencrypt/website/blob/master/docs/staging-environment.md#rate-limits)?